src/components/display/PhoneInput.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react"
+
+import {
+  InputGroup,
+  InputLeftAddon,
+  Input,
+  InputRightAddon,
+  Select,
+} from "@chakra-ui/react"
+
+export interface PhoneInputProps {
+  onChange: (phoneNumber: string) => void
+  defaultCountry?: "AU" | "US"
+}
+
+export const PhoneInput = (props: PhoneInputProps) => {
+  const { onChange, defaultCountry = "AU" } = props
+
+  const [country, setCountry] = React.useState(defaultCountry)
+  const [number, setNumber] = React.useState()
+
+  const isAustralia = country === "AU"
```

Comment:
right now we only want to support two countries, yep, but we should write our control flow to expect more than two. I think the way to go about this is by adding an enum and explicitly setting the type of `country` state to that enum.

once established we can also extract content (such as the emoji flags) which aren't really expected to ever change.

for example, something like this..

```typescript
export enum CountryName {
    AU = "Australia",
    US = "United States of America"
}

export type SelectableCountries = {
    [key in CountryName]: {
        callingCode: `+${string}`
        emoji: string
        examplePhoneNumber: string
    }
}

const countries: SelectableCountries = {
    [CountryName.AU]: {
        callingCode: "+61",
        emoji: "🇦🇺",
        examplePhoneNumber: "491 570 006"
    },
    [CountryName.US]: {
        callingCode: "+1",
        emoji: "🇺🇸",
        examplePhoneNumber: "555 123 4567"
    }
}
```

```typescript
const [selectedCountry, setSelectedCountry] = React.useState<CountryName>(initialCountryCode)
```

```tsx
<Select onChange={onCountryChange}>
  {Object.keys(countries).map((code) => {
    <option value={code} key={code}>
        {code}{" "}
        <span role="img" aria-label={CountryName[code]}>
          {countries.code.emoji}
        </span>
     </option>
  })}
</Select>
```